crates/edr_solidity_tests/src/multi_runner.rs

```diff
@@ -200,8 +203,6 @@ impl<
         .await
         .expect("Thread shouldn't panic")?;
 
-        let test_options: TestOptions = TestOptions { fuzz, invariant };
-
         if generate_gas_report {
             // Traces are needed to generate a gas report
             include_traces = IncludeTraces::All;
```

Comment:
Suggestion: The hack [introduced](https://github.com/NomicFoundation/edr/pull/1147/files#diff-bd0001d25e0dc485810c1b1e3f06130594c921cbe9a2fdfdd8200d4e7c041bd5R212) on line 213 should be resolved by this PR. As such, it would be good to remove the following lines:

```rs
        // HACK: When EVM isolation is enabled, we need to allow internal expectRevert
        // calls because we have a bug in isolation mode.
        if evm_opts.isolate {
            cheats_config_options.allow_internal_expect_revert = true;
        }
```

That way we can validate whether isolation mode runs correctly now.

Reply:
Merged